ouroboros-consensus-cardano/src/Ouroboros/Consensus/Cardano.hs

```diff
@@ -80,10 +83,10 @@ type ProtocolCardano        = HardForkProtocol '[ByronBlock, ShelleyBlock Standa
 data Protocol (m :: Type -> Type) blk p where
   -- | Run PBFT against the real Byron ledger
   ProtocolByron
```

Comment:
Introduce record for the Byron/Shelley/Allegra/Mary arguments and record for the per-transition argument (with the two eras as phantom type parameters).